### PR TITLE
Add Jdbc Select

### PR DESCRIPTION
**Pull request by JuHyun419**

### CODE REVIEW

File: dss-core/pom.xml [hunk 1]
```diff
@@ -84,5 +84,21 @@
...
+            <version>3.0.0</version>
+        </dependency>
+        
+        
+        <!-- logging -->
+        <dependency>
+			<groupId>log4j</groupId>
+			<artifactId>log4j</artifactId>
+			<version>1.2.17</version>
+		</dependency>
```
Comment (ztkmkoo) [inline dss-core/pom.xml]: please reformat here. thank you

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/blocking/jdbc/DssJdbcCommand.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import lombok.Setter;
+
+import java.util.List;
+import java.util.Map;
+
+
+public interface DssJdbcCommand extends DssCommand {
+
+    @Getter @Setter
+    class SelectResponse implements DssJdbcCommand {
```
Comment (ztkmkoo) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/blocking/jdbc/DssJdbcCommand.java]: need serialVersionUID here i think.

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/blocking/jdbc/DssJdbcCommand.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+        @Builder
+        public SelectResponse(String query) {
+            this.query = query;
+        }
+    }
+
+    @Getter @Setter
+    class Select implements  DssJdbcCommand, DssSelectCommand {
+
```
Comment (ztkmkoo) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/blocking/jdbc/DssJdbcCommand.java]: also need serialVersionUID here i think.

File: dss-core/pom.xml (hunk 1 above)
Comment (JuHyun419) [reply dss-core/pom.xml]: Okay, I will.

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/blocking/jdbc/DssSelectCommand.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+import io.github.ztkmkoo.dss.core.message.DssCommand;
+
+import java.util.Map;
+
+public interface DssSelectCommand<T> extends DssCommand {
+	
+//	String getNamespace();
+//	String getQueryId();
+//	T getEntity();
```
Comment (ztkmkoo) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/blocking/jdbc/DssSelectCommand.java:11]: I appreciate you removing this unusing comment~~

Comment (ztkmkoo) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/message/blocking/jdbc/DssSelectCommand.java]: I appreciate you clearing this unusing comments

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+import com.healthmarketscience.sqlbuilder.SelectQuery;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import io.github.ztkmkoo.dss.core.message.blocking.jdbc.DssJdbcCommand;
+import lombok.extern.log4j.Log4j;
+
+
+@Log4j
```
Comment (ztkmkoo) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java]: @JuHyun419 
Hi there,

you can log with context.getLog() in actor
and already we on dependency with logback ^^

Comment (JuHyun419) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java]: Oh I'll check!

Comment (JuHyun419) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java:14]: Oh I'll check :)

File: dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+	private Behavior<DssJdbcCommand> dssDatabaseServiceActor() {
+		return Behaviors
+				.receive(DssJdbcCommand.class)
+				.onMessage(DssJdbcCommand.Select.class, this::select)
+				.build();
+	}
+	
+    public static Behavior<DssJdbcCommand> create() {
+        return Behaviors.setup(context -> new DssDatabaseServiceActor(context).dssDatabaseServiceActor());
+    }
```
Comment (RulLu16) [inline dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java]: Hi, I think this static method goes to top. 
Ref https://github.com/ztkmkoo/dss/wiki/Code-Style(KR).

Comment (JuHyun419) [reply dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java]: Oh you're right! thank you :)

File: dss-core/src/test/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActorTest.java [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    void selectTest() throws IOException {
+        SelectQuery query = new SelectQuery()
+                .addCustomFromTable(testTable)
+                .addCustomColumns("name", "age")
+                .addCondition(BinaryCondition.equalTo("name", "JuHyun"))
+                .addCondition(BinaryCondition.equalTo("age", "10"))
+                .validate();
+        
+        String statement = query.toString();
+        System.out.println(statement);
```
Comment (ztkmkoo) [inline dss-core/src/test/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActorTest.java:37]: Hi, @JuHyun419 
Why not assert this statement?
Like assertEquals("SELECT name, age FROM test_table WHERE name = 'JuHyun' AND age = '10'", statement);
And also other test.

Comment (JuHyun419) [reply dss-core/src/test/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActorTest.java:37]: Okay~ I'll update 👍

**Comment by ztkmkoo** [comment]
@JuHyun419 
Hi there,
It looks some codes was conflicted. Could you resolve it?

**Comment by JuHyun419** [comment]
> @JuHyun419
> Hi there,
> It looks some codes was conflicted. Could you resolve it?

**Comment by JuHyun419** [comment]
> > @JuHyun419
> > Hi there,
> > It looks some codes was conflicted. Could you resolve it?

Okay~

**Comment by JuHyun419** [comment]
> @JuHyun419
> Hi there,
> It looks some codes was conflicted. Could you resolve it?

Okay~

**Pull request closed without merging**